ocean_lib/example_config.py

```diff
@@ -74,6 +74,18 @@
     },
 }
 
+NETWORK_IDS = {
+    8996: "development",
+    1: "mainnet",
+    5: "goerli",
+    56: "bsc",
+    137: "polygon",
+    246: "energyweb",
+    1287: "moonbase",
+    1285: "moonriver",
+    80001: "mumbai",
+}
+
```

Comment:
If you're going to insert NETWORK_IDs, it seems strange to put them into a file named "example_config.py". These aren't _examples_, this is basically hardcoded constants for dealing with networks.

The closest similar construct in df-py is in a file ["networkutil.py"](https://github.com/oceanprotocol/df-py/blob/main/util/networkutil.py). That name makes more sense.

Reply:
documented in https://github.com/oceanprotocol/ocean.py/issues/1019 as part of the example_config changes.